src/Resources/Locales/en.json

```diff
@@ -461,7 +461,11 @@
             "noData": "You haven't configured any Visual Rules yet",
             "addRuleButtonText": "Add Rule",
             "editRule": "Edit rule",
-            "removeRule": "Remove rule"
+            "removeRule": "Remove rule",
```

Comment:
we usually call it Delete instead of Remove in our app. Can we update that?

Reply:
really? i saw remove 
![image](https://user-images.githubusercontent.com/80120000/196562943-29735096-f131-4230-85ba-6d0e809dcee9.png)

Reply:
gahh. ok. You got me. Leave it.